src/lazy/encoder/annotate.rs

```diff
@@ -18,11 +18,11 @@ pub trait Annotatable {
     ///# use ion_rs::IonResult;
     ///# fn main() -> IonResult<()> {
     /// use ion_rs::{Element, IonData};
-    /// use ion_rs::lazy::encoder::text::LazyRawTextWriter_1_0;
+    /// use ion_rs::lazy::encoder::text::v1_0::writer::LazyRawTextWriter_1_0;
     /// use ion_rs::lazy::encoder::annotate::Annotatable;
     ///
     /// let mut buffer = vec![];
-    /// let mut writer = LazyRawTextWriter_1_0::new(&mut buffer);
+    /// let mut writer = LazyRawTextWriter_1_0::new(&mut buffer)?;
```

Comment:
🗺️ This was calling an inherent impl constructor on the LRTW before, but the `LazyRawWriter` trait expects `new` to return an `IonResult<Self>`. I removed the inherent impl in favor of the trait method to avoid that conflict.